app/build.gradle

```diff
@@ -1,306 +0,0 @@
-        vectorDrawables.useSupportLibrary = true
-        testInstrumentationRunner 'androidx.test.runner.AndroidJUnitRunner'
-
-        // The following argument makes the Android Test Orchestrator run its
-        // "pm clear" command after each test invocation. This command ensures
-        // that the app's state is completely cleared between tests.
-        // https://developer.android.com/training/testing/instrumented-tests/androidx-test-libraries/runner#use-android
-        // testInstrumentationRunnerArguments clearPackageData: 'true', coverage: 'true', coverageFilePath: '/data/data/ca.rmen.android.poetassistant.test/'
-
-
-        // used by Room, to test migrations
-        sourceSets {
-            main.java.srcDirs += "$projectDir/src/main/kotlin"
-            main.assets.srcDirs += "${project.buildDir}/generated/license_assets"
-            androidTest.assets.srcDirs +=
-                    files("$projectDir/src/androidTest/schemas".toString())
-            test {
-                manifest.srcFile "src/test/AndroidManifest.xml"
-                java.srcDirs += "$projectDir/src/test/kotlin"
-                java.srcDirs += "$projectDir/src/sharedTest/java"
-                java.srcDirs += "$projectDir/src/sharedTest/kotlin"
-            }
-            androidTest.java.srcDirs += "$projectDir/src/androidTest/kotlin"
-            androidTest.java.srcDirs += "$projectDir/src/sharedTest/java"
-            androidTest.java.srcDirs += "$projectDir/src/sharedTest/kotlin"
-        }
-
-        // used by Room, to test migrations
-        javaCompileOptions {
-            annotationProcessorOptions {
-                arguments = ["room.schemaLocation":
-                                     "$projectDir/schemas".toString()]
-            }
-        }
-    }
-
-    buildTypes {
-        debug {
-            testCoverageEnabled = project.gradle.startParameter.taskNames.any {
-                it.contains("jacocoTestReport")
-            }
-            applicationIdSuffix = ".test"
-            resValue "string", "search_provider_authority", android.defaultConfig.applicationId + applicationIdSuffix + ".SuggestionsProvider"
-        }
-        release {
-            minifyEnabled true
-            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
-            resValue "string", "search_provider_authority", android.defaultConfig.applicationId + ".SuggestionsProvider"
-        }
-    }
-
-    testOptions {
-        // Uncomment below to use orchestrator for tests
-        // execution 'ANDROIDX_TEST_ORCHESTRATOR'
-        unitTests {
-            all {
-                jvmArgs '-noverify', '-ea'
-                jacoco {
-                    includeNoLocationClasses = true
-                    jacoco.excludes = ['jdk.internal.*']
-                }
-            }
-            includeAndroidResources = true
-        }
-    }
-
-    if (rootProject.hasProperty("AndroidSigningKeyAlias")
```

Comment:
Had to move this up earlier, to before the signing configs are set for the release build type.